### ISSUE TITLE
Support Persisting Values

### ISSUE THREAD

**Issue by davelovesoftware** [bungeemonkee/Configgy#15]
Create a mechanism for saving changed values back to the persistence mechanism they came from.

Default to saving them to the source they came from, but ad an attribute to allow them to be saved to any specific source.

**Comment by vector-man** [comment]
So, does this mean Configgy does not currently support saving values?

**Comment by davelovesoftware** [comment]
Correct. I've been toying with possible pipelines that would be bidirectional but with the complexity of trying to figure out how to make the coercers/transformers reversible and the problem of "if you pull it from one of many sources which do you save it back to?" I haven't come up with any design I like yet.
